Connection.php

```diff
@@ -555,3 +555,26 @@ public function close($code = self::CLOSE_NORMAL, $reason = null)
         return $connection->disconnect();
     }
 }
+
+/**
+ * Register socket wrappers
```

Comment:
Or just `getSocketWrapper`, because the method is not responsible to register it, just to provide the “factory”.